docs/docs/guides/tasks.md

```diff
@@ -199,46 +199,72 @@ page.
-deployment workflow.
+The Keptn task context includes details about the current deployment, application name, version, object type and other
+user-defined metadata.
+Keptn populates this metadata while running tasks before and after deployments, to provide the necessary context
+associated with each task.
 
 This contrasts with the Kubernetes context, which is a set of access parameters that defines the
 specific cluster, user and namespace with which you interact.
-It serves as your personalized key to different areas within your Kubernetes infrastructure.
 For more information, see
 [Configure Access to Multiple Clusters](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
 
-Sometimes, to provide this crucial task context,
-you may need to embed it directly into the `function` code of your
-Keptn tasks defined in YAML files to define a
+For Tasks generated for applications running in Kubernetes, Keptn populates a `KEPTN_CONTEXT` variable containing a set
+of parameters that correlate a task
+to a specific stage, cluster or application.
+
+You can use this context information
+in the `function` code in your
 [KeptnTaskDefinition](../reference/crd-reference/taskdefinition.md)
 resource.
-The [keptn-tasks.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/keptn-tasks.yaml)
-file offers a practical example of how this works.
-
-A context environment variable is available via `Deno.env.get("KEPTN_CONTEXT")`.
-It can be used like this:
-
-```javascript
-let context = Deno.env.get("KEPTN_CONTEXT");
-    
-if (context.objectType == "Application") {
-    let application_name = contextdata.appName;
-    let application_version = contextdata.appVersion;
-}       
-        
-if (context.objectType == "Workload") {
-    let application_name = contextdata.appName;
-    let workload_name = contextdata.workloadName;
-    let workload_version = contextdata.workloadVersion;
-}
+
+By default, `KEPTN_CONTEXT` contains:
+
+- "appName"
+- "appVersion"
+- "workloadName"
+- "workloadVersion"
+- "taskType"
+- "objectType"
+- "metadata"
+
+A Job created by a Keptn Task with `KEPTN_CONTEXT`, may look like the following
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+spec:
+  template:
+    spec:
+      containers:
+        - name: "my-task-container"
+          env:
+            - name: KEPTN_CONTEXT
+              value: '{
+                         "workloadName":"waiter-waiter",
+                         "appName":"waiter",
+                         "appVersion":"",
+                         "workloadVersion":"0.4",
+                         "taskType":"pre",
+                         "objectType":"Workload",
+                         "metadata":{
+                            "commit-id":"1234",
+                            "stage":"dev",
+                            "test-metadata":"test-metadata"
+                         }
+                      }'
```

Comment:
the metadata will also contain a `traceparent` property that represents the span of the phase the KeptnTask is being executed in:

```suggestion
              value: '{
                         "workloadName":"waiter-waiter",
                         "appName":"waiter",
                         "appVersion":"",
                         "workloadVersion":"0.4",
                         "taskType":"pre",
                         "objectType":"Workload",
                         "metadata":{
                            "commit-id":"1234",
                            "stage":"dev",
                            "test-metadata":"test-metadata",
                            "traceparent": "00-traceid-spanid-01"
                         }
                      }'
```